### DISCUSSION TITLE
Adding Your Personal Blog to Linkfree: How to Do It Right?

### DISCUSSION THREAD

**Discussion by MuhammadNoorani** [EddieHubCommunity/LinkFree#6774] [Q&A] [answered]
As I am experiencing some confusion, I would greatly appreciate it if someone could clarify the process.  Also before there was a file of NOCODE-CHANGELOG.md for blogs and content, I can't find it.
Questions:

How can I add my personal blog to Linkfree?
What are the rules for adding a blog to Linkfree?
Are there any limitations on the type of content I can publish on my blog?
How can I make sure my blog aligns with the goals and values of the Linkfree project?

**Comment by eddiejaoude** [comment]
Hey @MuhammadNoorani 

> there was a file of NOCODE-CHANGELOG.md for blogs and content, I can't find it

this has been moved to a page within the app, to get it more visibility and renamed to `community resources` - we plan to do even more with this page soon https://linkfree.io/docs/community-resources (a link to this page is also in the footer)

> What are the rules for adding a blog to Linkfree?

What do you mean by "adding to LinkFree"? if to your profile, you can add any blog, if to the `community resources` page then it needs to be about LinkFree

> Are there any limitations on the type of content I can publish on my blog?

Not sure what you mean here? if to to the LinkFree page, please see my comment about and on the page itself (do we need to add more to the page to make it more clear?)

> How can I make sure my blog aligns with the goals and values of the Linkfree project?

Great question, what do you have in mind?

**Reply by MuhammadNoorani** [reply]
> What do you mean by "adding to LinkFree"? if to your profile, you can add any blog, if to the `community resources` page then it needs to be about LinkFree.

Yes, I was asking regarding the community resources part. So, my doubt is cleared if I have any content regarding Linkfree or related, it can be added to community resources.

> Not sure what you mean here? if to to the LinkFree page, please see my comment about and on the page itself (do we need to add more to the page to make it more clear?)

Yeah, I feel changes on the community page should be made, like if someone goes to the page and is writing content related to linkfree, so he knows that he can add it over there. 

And one more thing, I would like to propose, you can give the responisibilty to anyone for any updates, features added or any changes made to Linkfree is proposed in a blog also and on that page this will help project scale to more people.example: Updates in Linkfree - for the month May, Feature. This will help to document all updates, features and users to understand the project more. For this a seperate page for especially updates in form of blogs/threads can be added. This is my opinion.

> Great question, what do you have in mind?
 
I have in mind is that when I write content related to Linkfree or related and add it to the community resources, so what should that blog/thread or anything else be focused on?

**Reply by eddiejaoude** [reply, marked as answer]
> And one more thing, I would like to propose, you can give the responisibilty to anyone for any updates, features added or any changes made to Linkfree is proposed in a blog also and on that page this will help project scale to more people.example: Updates in Linkfree - for the month May, Feature. This will help to document all updates, features and users to understand the project more. For this a seperate page for especially updates in form of blogs/threads can be added. This is my opinion.

Thanks for the suggestion, if I understand correctly, we have this already https://linkfree.io/roadmap

> I have in mind is that when I write content related to Linkfree or related and add it to the community resources, so what should that blog/thread or anything else be focused on?

Anything related to LinkFree, we want to give the community a shoutout and support for their great content about LinkFree

**Reply by MuhammadNoorani** [reply]
Yeah, I just took a look at the roadmap part. It is exactly what I told. Amazing work.

All my doubts are cleared, Thanks for the answers Eddie!!

**Reply by eddiejaoude** [reply]
No problem, thanks for the discussion
